aiden-recommender/aiden_recommender/scrapers/france_travail/parser.py

```diff
@@ -0,0 +1,105 @@
+from aiden_recommender.scrapers.abstract_parser import AbstractParser
+from aiden_recommender.scrapers.field_extractors import (
+    FieldExtractorBase as feb,
+    MultipleFieldsExtractor as mfe,
+    NestedFieldExtractor as nfe,
+    FromMultipleFieldExtractor as fme,
+    FieldExtractorConstant as fce,
+    DateFieldExtractor as dfe,
+    ListFieldExtractor as lfe,
+)
+from aiden_recommender.models import Coordinates, Office, Organization, Profession
+import jmespath
+import re
+
+
+def parse_local_city(local_city):
+    local_city = local_city.split(" - ")
+    if len(local_city) > 1:
+        return local_city[1]
+    return None
+
+
+def parse_salary(data):
+    salary_str = jmespath.search("salaire.libelle", data)
+    if not salary_str:
+        return None, None, None
+    salary_values = re.findall(r"\d+(?:[.,]\d+)?", salary_str)
+    salary_values = [float(value.replace(",", ".")) for value in salary_values]
+
+    if "Horaire" in salary_str:
+        salary_period = "hour"
+    elif "Mensuel" in salary_str:
+        salary_period = "month"
+    elif "Annuel" in salary_str:
+        salary_period = "year"
+    else:
+        salary_period = "year"
+
+    if len(salary_values) == 1:
+        min_salary = max_salary = int(salary_values[0])
+    elif len(salary_values) >= 2:
+        min_salary = int(salary_values[0])
+        max_salary = int(salary_values[1])
+    else:
+        min_salary = max_salary = None
+
+    return min_salary, max_salary, salary_period
+
+
+def parse_experience_level(experience_level):
+    try:
+        return float(experience_level.split()[0])
+    except Exception:
+        return experience_level
+
+
+class FranceTravailParser(AbstractParser):
```

Comment:
@Arthtanc, what do you think of the selectors like this in aiden? Anything you would recommand to do differently?

Reply:
I will have to read this more carefully, but in general I would say that copying property-pipeline is a pretty bad idea, that repo is my reference on what not to do.

First of all, I am not a fan of using inheritance, and this design is essentially a whole genealogy tree of classes,
if it takes three or four classes to figure it out what a simple extractor does it is probably bad design (in my opinion).

I would rather have a simple function that is very explicit and easy to read and modify. But if the current code works, honestly you can leave it like that.

Reply:
Another thing, I think it would be better to leave the classes full name, it is more explicit and easy to read.

Reply:
I changed everything to make you happy @ArthTanc , tell me what you think now